next.js/BUILD.bazel

```diff
@@ -1,121 +1,10 @@
-load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")
-load("@npm//:next/package_json.bzl", next_bin = "bin")
-load("@npm//:defs.bzl", "npm_link_all_packages")
-load("@aspect_rules_js//js:defs.bzl", "js_binary", "js_run_binary", "js_test")
 load("@aspect_rules_ts//ts:defs.bzl", "ts_config")
+load("@npm//:defs.bzl", "npm_link_all_packages")
 
 npm_link_all_packages(name = "node_modules")
```

Comment:
is this used?

Reply:
yes. there is a pnpm workspace so the virtual store is linked created at root as well as some of devDependencies